src/pybind/mgr/dashboard/frontend/src/app/core/error/error.component.html

```diff
@@ -0,0 +1,36 @@
+<div *ngIf="route.data | async as data">
+  <div class="horizontal-align vertical-align full-height"
+       *ngIf="data.errorMessage">
+    <div class="text-center">
+      <h1 i18n>{{ data.errorMessage }}</h1>
```

Comment:
The last info I had was that variables can't be correctly translated. @tspmelo what's your take on that?

But you can do it anyway - if you just translate the strings beforehand.

Reply:
Thanks @Devp00l .! Will take a look